src/main/java/uk/gov/hmcts/reform/prl/enums/serviceofapplication/TransferOfCaseToAnotherCourtEnum.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.prl.enums.serviceofapplication;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.RequiredArgsConstructor;
+import uk.gov.hmcts.reform.prl.enums.CustomEnumSerializer;
+
+@RequiredArgsConstructor
+@JsonSerialize(using = CustomEnumSerializer.class)
+public enum TransferOfCaseToAnotherCourtEnum {
+
+    transferOfCaseToAnotherCourt("Transfer of case to another court (C49)");
+
+    private final String displayedValue;
+
+    @JsonValue
+    public String getDisplayedValue() {
+        return displayedValue;
+    }
+
+    @JsonCreator
+    public static TransferOfCaseToAnotherCourtEnum getValue(String key) {
+        return TransferOfCaseToAnotherCourtEnum.valueOf(key);
+    }
```

Comment:
We dont need this as C49 requirement has been withdrawn

Reply:
@BharadwajSairam - can you take a look at this and remove from your service of application implementation?